attic/05-extending.Rmd

```diff
@@ -251,7 +251,7 @@ PipeOpScaleAlways = R6::R6Class("PipeOpScaleAlways",
 )
 ```
 
-_(Note for the observant: If you check `PipeOpScale.R` from the `mlr3pipelines` package, you will notice that is uses "`get("type")`" and "`get("id")`" instead of "`type`" and "`id`", because the static code checker on CRAN would otherwise complain about references to undefined variables. This is a "problem" with `data.table` and not exclusive to `mlr3pipelines`.)_
+_(Note for the observant: If you check `PipeOpScale.R` from the `mlr3pipelines` package, you will notice that is uses "`get("type`" and "`get("id`" instead of "`type`" and "`id`", because the static code checker on CRAN would otherwise complain about references to undefined variables. This is a "problem" with `data.table` and not exclusive to `mlr3pipelines`.)_
```

Comment:
AFAICT this was okay and now looks wrong.

Reply:
Good catch, leftovers of the `sed` call I did. 
I checked most files quickly and thought all is fine.